DIPs/DIP1022.md

```diff
@@ -274,7 +271,14 @@ void main()
 
 - <a name="aliasseq"></a>specification of iteration over alias sequences:
 	* https://dlang.org/spec/statement.html#foreach_over_tuples
+	
+## Breaking Changes And Deprecations
 
+This DIP will deprecate an unknown amount of code. Every use of `foreach(ref <...>)`
```

Comment:
The DIP will *break* an unknown amount of code.  Features get deprecated, not user code.

On the same note, instead of "Every use of" should be "The use of".

Anyway, I can touch this up locally.

Reply:
Okay, I trust you to handle that.